main.py

```diff
@@ -2171,6 +2171,10 @@ def on_query_completions(self, prefix, locations):
             elif self.state == CompletionState.APPLYING:
                 self.state = CompletionState.IDLE
 
+            if self.state != CompletionState.IDLE:
+                debug("requesting completions")
+                self.view.run_command('hide_auto_complete')
```

Comment:
@randy3k : Can you check, whether removing this line fixes your issue? It is in the end the only change I could imagine to cause completions not to be displayed at all under some circumstances.

Reply:
Note: This line here hides the completion panel until the completions of a language server have arrived.